app/views/layouts/demo-layout.html

```diff
@@ -15,7 +15,7 @@
  limitations under the License.
 -->
 
-<html lang="en">
+<html lang="{{#unless data.options.lang}}en{{else}}{{data.options.lang}}{{/unless}}">
```

Comment:
I don't think this should work right now, should it? We have this pr in progress that will be able to make use of it https://github.com/subscriptions-project/swg-js/pull/3527

Reply:
I was able to get the prompts to render in a specified language only if I changed my browser setting. You are right though, this should work eventually, hopefully after your referenced PR is merged.